YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java

```diff
@@ -58,7 +59,8 @@
     WebviewOverlay fh;
     private boolean willStartMic = false;
     public String postUrl = "https://app.yellowmessenger.com/api/chat/upload?bot=";
-
+    private String updateUserStatusUrl = "https://app.yellowmessenger.com/api/presence/usersPresence/log_user_profile";
```

Comment:
how will it work in case of onPrem deployment?

Reply:
Will change this